tests/src/Util/HTTPInputDataTest.php

```diff
@@ -39,13 +39,26 @@ class HTTPInputDataTest extends MockedTest
 	 */
 	public function dataStream()
 	{
+		$_SERVER['CONTENT_TYPE'] = 'multipart/form-data;boundary=43395968-f65c-437e-b536-5b33e3e3c7e5;charset=utf8';
```

Comment:
plz move this into the test directly, `dataStream` is just for input values.
And maybe we do need a check if there's already a value set and reset it after the test

if you ever need to have different `CONTENT_TYPE` values for a test set. you could add it as a third value under `expected`, add a third parameter at the test and set it directly inside the test :-)

Reply:
This value for the boundary has to correspond with the file content. And since we set the file name here, I think we should set the value here as well, shouldn't we?

Reply:
looks good 2 me! :-)